frontend/src/App.tsx

```diff
@@ -10,12 +10,14 @@ import {
   ThemeProvider as MuiThemeProvider,
```

Comment:
dev branch has changed since this checkout, so I'm going to cherry pick (manually) the changes from this file.

I'm Closing this PR and applying the changes manually :)